actions/cursor-deploy/main.test.ts

```diff
@@ -10,7 +10,7 @@ jest.mock('@actions/github')
 const mockedUtils = jest.mocked(utils)
 
 describe(`Cursor Deploy Action`, () => {
-    test(
+    it(
```

Comment:
Actually these `test` was better as they describe a test case. 🙃 

So `test` for test case and `it` for sentences that reads well `it sanitizes` vs `test when the action runs`.

Reply:
word